### PR TITLE
typo correct: IBM Cloud is the right name

### PR DESCRIPTION
**Pull request by xunpan**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @jichenjc @xunpan

**Special notes for your reviewer**:

/sig ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ kubectl --kubeconfig=kubeconfig get machines -o yaml
 ### Cluster Deletion
 
 This guide explains how to delete all resources that were created as part of
-your ibmcloud Cluster API Kubernetes cluster.
+your IBMi Cloud Cluster API Kubernetes cluster.
```
Comment (gyliu513) [inline README.md]: s/IBMi Cloud/IBM Cloud

Comment (jichenjc) [reply README.md]: +1

Comment (xunpan) [reply README.md]: ^_^!!!

**Pull request merged**